gbpservice/nfp/configurator/drivers/loadbalancer/v2/haproxy/haproxy_driver.py

```diff
@@ -0,0 +1,592 @@
+# Copyright 2014, Doug Wiegley (dougwig), A10 Networks
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import ast
+import copy
+
+from neutron._i18n import _LI
+from neutron_lbaas.drivers import driver_base as n_driver_base
+from oslo_log import log as logging
+
+from gbpservice.nfp.common import exceptions
+from gbpservice.nfp.configurator.drivers.base import base_driver
+from gbpservice.nfp.configurator.drivers.loadbalancer.v1.haproxy.\
```

Comment:
@jiahaoliang Initially I thought of suggesting to keep LbGenericConfigDriver code in separate file but what I thought is going ahead when LBV1 won't be there no one else will be importing this file except LBV2, so it becomes LBV2 agent specific only. So better to copy the code.